src/sif/dynamiccost.cc

```diff
@@ -452,6 +452,11 @@ void ParseBaseCostOptions(const rapidjson::Value& value,
   // closure_factor
   pbf_costing_options->set_closure_factor(base_cfg.closure_factor_(
       rapidjson::get<float>(value, "/closure_factor", base_cfg.closure_factor_.def)));
+
+  // HOT/HOV
+  pbf_costing_options->set_use_hot(rapidjson::get<bool>(value, "/use_hot", base_cfg.use_hot_));
+  pbf_costing_options->set_use_hov2(rapidjson::get<bool>(value, "/use_hov2", base_cfg.use_hov2_));
+  pbf_costing_options->set_use_hov3(rapidjson::get<bool>(value, "/use_hov3", base_cfg.use_hov3_));
```

Comment:
i think we should use different nomenclature. all of the other options which are prefixed with `use_` are a sliding scale. this is a boolean option. we should name it something else to remove that confusion or unintuitiveness. maybe `include_` or something to match the include/exclude language used in other boolean flags and attribute filters?

Reply:
maybe we should just use `"hot": true, "hov2": true, "hov3": true` and avoid the extra verbage all together?